src/main/java/edu/kis/powp/jobs2d/drivers/SelectDriverMenuOptionListener.java

```diff
@@ -17,5 +18,5 @@ public SelectDriverMenuOptionListener(Job2dDriver driver, DriverManager driverMa
     @Override
     public void actionPerformed(ActionEvent e) {
         driverManager.setCurrentDriver(driver);
-    }
+        DriverFeature.updateDriverInfo();}
```

Comment:
This line was removed in other PR, and it should not be here! Some problems in merging changes?!